_config.yml

```
@@ -3,7 +3,7 @@ title: Ottawa Civic Tech
 hashtag: "#YOWCT"
 sub_title: Build civic good.
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "" # the base hostname & protocol for your site, e.g. http://example.com
+url: "ottawacivictech.ca" # the base hostname & protocol for your site, e.g. http://example.com
```

Not sure if this should match the CNAME
```suggestion
url: "www.ottawacivictech.ca" # the base hostname & protocol for your site, e.g. http://example.com
```

---

Sure. fixed.